qml/ConnectionMenu.qml

```diff
@@ -81,39 +82,54 @@ Item {
             columnSpacing: 5
 
             Text {
-                text: "Communication:"
+                text: "Device:"
                 enabled: true
                 color: Material.primary
             }
 
             ComboBox {
+                id: deviceCB
+                Layout.fillWidth: true
+                textRole: "name"
+                //TODO: This model should be provided by someone else and not to be hardcoded here
```

Comment:
you can perhaps use the QMetaType information of the menu to retrieve the values, that are generated by Q_ENUM to fill a list model. but that can be in another patch.

Reply:
The problem here is not the enum, but the map between string and enum.